app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitSelectionPlansApiController.php

```diff
@@ -327,6 +336,64 @@ public function getCurrentSelectionPlanByStatus($summit_id, $status)
 
     use ParametrizedGetAll;
 
+    /**
+     * @param $summit_id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function getAllCurrentSelectionPlansByStatus($summit_id)
+    {
+        try {
+            return $this->_getAll(
+                function () {
+                    return [
+                        'status' => ['=='],
+                    ];
+                },
+                function () {
+                    return [
+                        'status' => 'sometimes|string|in:submission,selection,voting',
+                    ];
+                },
+                function () {
+                    return [
+                        'id',
+                        'created',
+                        'last_edited',
+                    ];
+                },
+                function ($filter) use ($summit_id) {
+                    if ($filter instanceof Filter) {
+                        $filter->addFilterCondition(FilterElement::makeEqual('summit_id', $summit_id));
+                    }
+                    return $filter;
+                },
+                function () {
+                    return SerializerRegistry::SerializerType_Public;
+                },
+                null,
+                null,
+                function ($page, $per_page, $filter, $order, $applyExtraFilters) {
```

Comment:
this is not necessary
the default implementation of the trait already calls this
https://github.com/OpenStackweb/summit-api/blob/f0515686ccf1d21fbaa0691fab78e5aecba89f75/app/Http/Controllers/Apis/Protected/Summit/Traits/ParametrizedGetAll.php#L45

Reply:
This was changed as requested